pandas/tests/indexes/test_base.py

```diff
@@ -29,6 +29,24 @@
 from pandas._libs.lib import Timestamp
 
 
+@pytest.fixture(params=[tm.makeUnicodeIndex(100),
+                        tm.makeStringIndex(100),
```

Comment:
move these to a conftest.py in this directory then they can be used in any test files